_posts/en/newsletters/2025-03-14-newsletter.md

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Bitcoin Optech Newsletter #345'
+permalink: /en/newsletters/2025/03/14/
+name: 2025-03-14-newsletter
+slug: 2025-03-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **P2P traffic analysis:** developer Virtu [posted][virtu traffic] to
+  Delving Bitcoin an analysis of the network traffic generated and
+  received by his node in four different modes: initial block download
+  (IBD), non-listening (outbound connections only), non-archival (pruned) listening, and archival
+  listening.  Although the results for his single node may not have been
+  representative in all cases, we found several of his discoveries to by
+  interesting:
+
+  - *High block traffic as an archival listening node:* Virtu's node served
+    several gigabytes of blocks each hour to other nodes when operating
+    as a non-pruned listening node.  Many of the blocks were older blocks
+    being requested by incoming connections so that they could perform
+    IBD.
+
+  - *High inv traffic as a non-archival listener:* about 20% of total node
+    traffic was `inv` messages before he enabled serving of older
+    blocks.  [Erlay][topic erlay] may significantly reduce
+    that 20% overhead, which represented about 100 megabytes a day.
+
+  - *Bulk of inbound peers appear to be spy nodes:* "Interestingly, the
+    bulk of inbound peers exchange only around 1MB of traffic with my
+    node, which is too low (using traffic via my outbound connections as
+    baseline) for them to be regular connections. All those nodes do is
+    complete the P2P handshake, and politely reply to ping messages.
+    Other than that, they just suck up our `inv` messages."
+
+  Virtu's post contains additional insights and multiple charts
+  illustrating the traffic experienced by his node.
+
+- **Research into single-path LN pathfinding:** Sindura Saraswathi
+  [posted][saraswathi path] to Delving Bitcoin about [research][sk path]
+  she conducted with Christian Kümmerle about finding optimal paths
+  between LN nodes for sending payments in a single part.  Her post
+  describes the strategies currently used by Core Lightning, Eclair,
+  LDK, and LND.  The authors then use eight modified and unmodified LN
+  nodes in a simulated LN network (based on a snapshot of the actual
+  network) to test pathfinding, evaluating based criteria such as
```

Comment:
```suggestion
  network) to test pathfinding, evaluating criteria such as
```

The word ‘based’ does not seem to belong. Or was it supposed to be:

```suggestion
  network) to test pathfinding, evaluating basic criteria such as
```

Reply:
It was supposed to be "based on", but your suggestion to drop "based" entirely is best of all.